helper-scripts/README.md

```diff
@@ -1,12 +1,34 @@
 # Helper scripts
 
-## delete_stack.sh
+## `delete_stack.sh`
 
 This script provides the ability to:
+
 - empty and delete any versioned S3 buckets (if present in a SAM application)
 - delete the SAM application
 
 ```bash
 export STACK_NAME=backend-stack-name    (REQUIRED)
 ./delete_stack.sh
-```
\ No newline at end of file
+```
+
+## `deploy_backend.sh`
+
+### What it does
+
+This script provides the ability to deploy a `backend-api` stack in the AWS dev account.
+
+### How to use it
+
+#### Execute `deploy-backend-to-dev` script
+
+```bash
+# From /backend-api
+npm run deploy-backend-to-dev <your-stack-name>
+```
+
+#### Follow the prompts
+
+1. Deploying an `sts-mock` stack. This is required the first time you deploy a `backend-api` stack. It's optional for subsequent deployments.
```

Comment:
Is it technically required? I think this guidance should be in the echo statements from the script rather than the readme potentially?

Reply:
Active Session API tests will fail if it's not deployed so to have a full working `backend-api` I believe it is required. Do you think otherwise? I suppose you could argue if you're testing asyncToken or asyncCredential in isolation it wouldn't be required but quite niche to put in a guide.

Will see what it looks like with guidance echoed in terminal now..

Reply:
I wonder if we should just always deploy both stacks here and simplify the user inputs? If it's painful, we can add the checks in?

Reply:
To run the sts api tests, you need the backend. If we have environment coupling it feels sensible to deploy both at once

Reply:
Will iterate in future as discussed